modules/ftrackIdSystem.md

```diff
@@ -0,0 +1,90 @@
+# Flashtalking's FTrack Identity Framework User ID Module
+
+*The FTrack Identity Framework User ID Module allows publishers to take advantage of Flashtalking's FTrack ID during the bidding process.*
+
+**THE ONLY COMPLETE SOLUTION FOR COOKIELESS MEASUREMENT & PERSONALIZATION**
+
+Flashtalking is the world’s first ad serving platform to function without cookies to orchestrate client identity across buy-side ID spaces for measurement and personalization. With over 120 active global advertisers, our cookieless identity framework is market-ready and includes privacy controls to ensure consumer notification and choice on every impression.
```

Comment:
remove marketing language. this is not an advertisement, it should simply describe your module.

Reply:
Removed